### PR TITLE
lci: upstream v1.0 release

### PR DESCRIPTION
**Pull request by chatziko**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/lci.rb [hunk 1]
```diff
@@ -16,10 +16,18 @@ class Lci < Formula
...
 
+  depends_on "cmake" => :build
+
   conflicts_with "lolcode", because: "both install `lci` binaries"
 
   def install
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "install"
+    mkdir "build" do
+      system "cmake", "..", "-DCMAKE_BUILD_TYPE=Release", "-DCMAKE_INSTALL_PREFIX=#{prefix}"
```
Comment (carlocab) [inline Formula/lci.rb:25]:
```suggestion
      system "cmake", "..", *std_cmake_args
```

File: Formula/lci.rb [hunk 2]
```diff
@@ -16,10 +16,18 @@ class Lci < Formula
...
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match "[I, 2]", shell_output("echo 'Append [1] [2]' | #{bin}/lci")
```
Comment (carlocab) [inline Formula/lci.rb:31]:
```suggestion
    assert_match "[I, 2]", pipe_output("#{bin}/lci", "Append [1] [2]")
```

**Review by carlocab — COMMENTED** [review]
Thanks, @chatziko. Just a few suggestions.

**Comment by chatziko** [comment]
> Thanks, @chatziko. Just a few suggestions.

Thanks, they're committed.

**Review by carlocab — APPROVED** [review]
Thanks again, @chatziko.

**Pull request closed without merging**